L1Trigger/TrackTrigger/src/StubPtConsistency.cc

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include <memory>
+
+#include "L1Trigger/TrackTrigger/interface/StubPtConsistency.h"
+
+namespace StubPtConsistency {
+
+  float getConsistency(TTTrack< Ref_Phase2TrackerDigi_ > aTrack, const TrackerGeometry* theTrackerGeom, const TrackerTopology* tTopo, double mMagneticFieldStrength, int nPar) {
+    double trk_bendchi2 = 0.0;
+
+    if ( !(nPar==4 || nPar==5)) {
+      std::cerr << "Not a valid nPar option!" << std::endl;
```

Comment:
`std:cerr` is not allowed (not threadsafe)

It looks like this should case issue an exception rather than returning a default value.